WorkloadManagementSystem/Service/WMSAdministratorHandler.py

```diff
@@ -64,11 +64,18 @@ def export_setSiteMask( self, siteList ):
     return result
 
 ##############################################################################
-  types_getSiteMask = []
-  def export_getSiteMask(self):
+  types_getSiteMask = [basestring]
+  def export_getSiteMask(self, siteState = 'Active'):
     """ Get the site mask
     """
-    return jobDB.getSiteMask( 'Active' )
+    return jobDB.getSiteMask( siteState )
+
+##############################################################################
+  types_getSiteMaskStatus = [basestring]
+  def export_getSiteMaskStatus(self, sites = None):
```

Comment:
BTW, we need a integration test for this function too, to be added to Test_Client_WMS.py file